### PR TITLE
add DELETE permission on placementdecision

### PR DESCRIPTION
**Pull request by elgnay**
The related issue: https://github.com/open-cluster-management/backlog/issues/14538
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: pkg/operators/clustermanager/bindata/bindata.go [hunk 1]
```diff
@@ -1676,8 +1676,11 @@ rules:
   verbs: ["get", "list", "watch"]
 # Allow controller to manage placements/placementdecisions
 - apiGroups: ["cluster.open-cluster-management.io"]
-  resources: ["placements", "placementdecisions"]
+  resources: ["placements"]
   verbs: ["get", "list", "watch", "create", "update", "patch"]
```
Comment (qiujian16) [inline pkg/operators/clustermanager/bindata/bindata.go:1680]: I think you need to remove create in placement

Comment (elgnay) [reply pkg/operators/clustermanager/bindata/bindata.go]: 'create' removed.

File: manifests/cluster-manager/cluster-manager-placement-clusterrole.yaml [hunk 2]
```diff
@@ -17,8 +17,11 @@ rules:
   verbs: ["get", "list", "watch"]
 # Allow controller to manage placements/placementdecisions
 - apiGroups: ["cluster.open-cluster-management.io"]
-  resources: ["placements", "placementdecisions"]
-  verbs: ["get", "list", "watch", "create", "update", "patch"]
+  resources: ["placements"]
+  verbs: ["get", "list", "watch", "update", "patch"]
```
Comment (qiujian16) [inline manifests/cluster-manager/cluster-manager-placement-clusterrole.yaml:21]: I do no think it needs update/patch either.

**Pull request merged**